projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/components/query-check-your-answers/query-check-your-answers.component.html

```diff
@@ -1,12 +1,14 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds-from-desktop">
-    <h1 class="govuk-heading-l">{{ 'Review query response details' | rpxTranslate }}</h1>
+    <h1 class="govuk-heading-l">
+      {{ 'Review query details' | rpxTranslate }}
+    </h1>
 
     <div class="govuk-!-margin-bottom-4">
       <ccd-query-case-details-header></ccd-query-case-details-header>
     </div>
 
-    <dl class="govuk-summary-list govuk-!-margin-bottom-0">
+    <dl *ngIf="context === 'Responded'" class="govuk-summary-list govuk-!-margin-bottom-0">
```

Comment:
@DavidJayakumar - similar to @MateiHMCTS's comment, please use the enum value instead of the string `'Responded'`